src/popup.html

```diff
@@ -1,170 +1,304 @@
-          <h1 class="settings-hl i18n-content" data-i18n-message-id="popupSettingsTitle"></h1>
-        </div>
-        <ul class="settings-list">
-          <li class="setting">
-            <span class="setting-label flex-col i18n-content" data-i18n-message-id="popupSettingsShowIcon"></span>
-            <button title="" data-icon-visibility-option="disable-input-icon" class="toggle-icon-in-page-visibility fx-relay-toggle"></button>
-          </li>
-          <li class="setting">
-            <span class="setting-label flex-col i18n-content" data-i18n-message-id="popupSettingsCollectData"></span>
-            <button title="" class="data-collection fx-relay-toggle"></button>
-          </li>
-          <li class="setting flex-col">
-            <a id="popupSettingsLeaveFeedbackLink" href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay-addon&utm_medium=popup" class="setting-label setting-link close-popup-after-click i18n-content" data-event-action="click" data-event-label="panel-leave-feedback-link" data-i18n-message-id="popupSettingsLeaveFeedback"></a>
-            <a href="https://www.mozilla.org/privacy/firefox-relay/?utm_source=fx-relay-addon&utm_medium=popup" class="setting-label setting-link close-popup-after-click i18n-content" data-event-action="click" data-event-label="panel-privacy-link" data-i18n-message-id="popupSettingsPrivacy"></a>
-            <a href="https://www.mozilla.org/about/legal/terms/firefox-relay/?utm_source=fx-relay-addon&utm_medium=popup" class="setting-label setting-link close-popup-after-click i18n-content" data-event-action="click" data-event-label="panel-terms-of-service" data-i18n-message-id="popupSettingsTermsOfService"></a>
-          </li>
-        </ul>
-      </settings>
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8">
+  <title>Firefox Relay</title>
+  <link rel="stylesheet" href="css/relay.css">
+  <link rel="stylesheet" href="css/popup.css">
+  <link rel="stylesheet" href="css/global.css">
+  <script src="/js/libs/browser-polyfill.min.js"></script>
+  <script src="/js/shared/i18n.js"></script>
+  <script src="/js/shared/data-opt-out-toggle.js"></script>
+  <script src="js/shared/utils.js"></script>
+  <script src="js/popup/popup.js"></script>
+  <script src="js/shared/metrics.js"></script>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
 
-      <sign-up-panel class="sign-up-panel hidden">
-        <span class="welcome i18n-content" data-i18n-message-id="popupSignUpPanelWelcome"></span>
-        <p class="intro i18n-content" data-i18n-message-id="popupSignUpPanelIntro"></p>
-        <a class="button sign-in-btn blue-primary-btn close-popup-after-click login-link i18n-content" data-event-action="click" data-event-label="panel-continue-btn" data-i18n-message-id="popupSignUpPanelContinue" href=""></a>
-      </sign-up-panel>
+<body class="fx-relay-panel fx-relay">
```

Comment:
issue (blocking): Following up on @Vinnl comment about this diff being big. I was able to hide whitespace on this review and noticed you have nested `<body><head>` tags below. Seems like maybe a bad rebase? 

Can you review / reset this file and contain your changes to `<report-issue>`

Reply:
Oh wow, thank you for pointing that out. I didn't even notice any visible changes to the panel with this error which is weird.